linajea/config/evaluate.py

```diff
@@ -11,17 +11,6 @@
 class _EvaluateParametersConfig:
     matching_threshold = attr.ib(type=int)
     roi = attr.ib(converter=ensure_cls(DataROIConfig), default=None)
-    # deprecated
-    frames = attr.ib(type=List[int], default=None)
-    # deprecated
-    frame_start = attr.ib(type=int, default=None)
-    # deprecated
-    frame_end = attr.ib(type=int, default=None)
-    # deprecated
-    limit_to_roi_offset = attr.ib(type=List[int], default=None)
-    # deprecated
-    limit_to_roi_shape = attr.ib(type=List[int], default=None)
-    # deprecated
```

Comment:
I also thought about this yesterday. Maybe we should leave this in for a little bit, I might have some older databases that use it.

Reply:
Cool, I can just revert that commit, and maybe add a warning about deprecation